staging/src/k8s.io/apiserver/pkg/admission/plugin/resourcequota/controller.go

```diff
@@ -518,11 +518,9 @@ func CheckRequest(quotas []corev1.ResourceQuota, a admission.Attributes, evaluat
 			if innerErr != nil {
 				return quotas, innerErr
 			}
-			if feature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScaling) {
+			if feature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScaling) && a.GetResource().GroupResource() == corev1.Resource("pods") {
```

Comment:
> That said, I don't think it's correct to allow negative deltas in a request like this, since admission can complete but persistence in etcd can fail. That would allow a request to decrement quota without actually reducing use.

As another concrete example of this, we don't decrement count quota when deleting an object, we wait for the controller to observe the absence of the object and free the count quota